_posts/2016-10-24-6.18.0.md

```diff
@@ -0,0 +1,331 @@
+  }
+}
+```
+
+[#4655](https://github.com/babel/babel/pull/4655) Add `useBuiltIns` option to helper-builder-react-jsx. ([@existentialism](https://github.com/existentialism))
+
+Previously we added a `useBuiltIns` for object-rest-spread so that it use the native/built in version if you use a polyfill or have it supported natively.
+
+This change just uses the same option from the plugin to be applied with spread inside of jsx.
+
+```js
+// in
+var div = <Component {...props} foo="bar" />
+// out
+var div = React.createElement(Component, Object.assign({}, props, { foo: "bar" }));
+```
+
+[#4724](https://github.com/babel/babel/pull/4724) Add `EmptyTypeAnnotation`. ([@samwgoldman](https://github.com/samwgoldman))
+
+Added in flow [here](https://github.com/facebook/flow/commit/c603505583993aa953904005f91c350f4b65d6bd) and in babylon [here](https://github.com/babel/babylon/pull/171).
+
+```js
+function f<T>(x: empty): T {
+  return x;
+}
+f(); // nothing to pass...
+```
+
+[#4758](https://github.com/babel/babel/pull/4758) Make getBinding ignore labels; add Scope#getLabel, Scope#hasLabel, Scope#registerLabel. ([@kangax](https://github.com/kangax))
+
+Track `LabeledStatement` separately (not part of bindings).
+
+## 💅 Polish
+[#4690](https://github.com/babel/babel/pull/4690) Consolidate contiguous var declarations in destructuring transform. ([@motiz88](https://github.com/motiz88))
+
+```js
+// in
+const [a, b] = [1, 2];
+// out
+var a = 1,
+    b = 2;
+```
+
+[#4685](https://github.com/babel/babel/pull/4685) Better error messaging when preset options are given without a corresponding preset. ([@kaicataldo](https://github.com/kaicataldo))
+
+> We've had a few reports of users not wrapping a preset in `[]` when passing in options so we added an extra error message for this.
+
+```js
+// incorrect, and current gives a unexpected error message
+{
+  "presets": [
+    "preset",
+    { "presetOptions": 'hi' } // gets parsed as another preset instead of being part of the "preset"
+  ]
+}
+// correct
+{
+  "presets": [
+    ["preset",
+      {
+        "presetOptions": 'hi'
+      }
+    ]
+  ]
+}
+```
+
+```
+ReferenceError: [BABEL] /test.js: Unknown option: base.loose2. Check out http://babeljs.io/docs/usage/options/ for more information about options.
+
+A common cause of this error is the presence of a configuration options object without the corresponding preset name. Example:
+
+Invalid:
+  `{ presets: [{option: value}] }`
+Valid:
+  `{ presets: ["pluginName", {option: value}] }`
+
+For more detailed information on preset configuration, please see http://babeljs.io/docs/plugins/#pluginpresets-options.
+```
+
+## 📝 Documentation
+[#4653](https://github.com/babel/babel/pull/4653) Tweak license for GitHub display. ([@existentialism](https://github.com/existentialism))
+
+So that our MIT License [shows up](https://github.com/blog/2252-license-now-displayed-on-repository-overview).
+
+## 🐛 Bug Fixes
+
+> Will give examples of code that was fixed below
```

Comment:
Is this copy supposed to be here, or just leftovers from a template for the release notes?